e2e/cypress/integration/channel/message_reply_bot_post_spec.js

```diff
@@ -44,22 +46,70 @@ describe('Message Reply with attachment pretext', () => {
-
                 // # Post message with auth token
-                const message = 'Hello message from ' + botName;
-                const props = {attachments: [{pretext: 'Some Pretext', text: 'Some Text'}]};
-                cy.postBotMessage({token, message, props, channelId: newChannel.id, createAt: yesterdaysDate}).
+                const props = {attachments: [{text: 'Some Text posted by bot that has no content and no attachment pretext'}]};
+                cy.postBotMessage({token: botToken, props, channelId: newChannel.id, createAt: yesterdaysDate}).
                     its('id').
                     should('exist').
-                    as('yesterdaysPost');
+                    as('yesterdaysBotPost');
+            });
+        });
+
+        // # Add two subsequent posts
+        cy.postMessage('First posting');
+        cy.postMessage('Another one Posted');
+
+        cy.get('@yesterdaysBotPost').then((postId) => {
+            // # Open RHS comment menu
+            cy.clickPostCommentIcon(postId);
+
+            // # Reply to message
+            cy.postMessageReplyInRHS('A reply to an older post bot post');
+
+            // # Get the latest reply post
+            cy.getLastPostId().then((replyId) => {
+                // * Verify that the reply is in the RHS with matching text
+                cy.get(`#rhsPost_${replyId}`).within(() => {
+                    cy.queryByTestId('post-link').should('not.be.visible');
+                    cy.get(`#rhsPostMessageText_${replyId}`).should('be.visible').and('have.text', 'A reply to an older post bot post');
+                });
+
+                cy.get(`#CENTER_time_${postId}`).find('time').invoke('attr', 'title').then((originalTimeStamp) => {
+                    // * Verify the first post timestamp equals the RHS timestamp
+                    cy.get(`#RHS_ROOT_time_${postId}`).find('time').invoke('attr', 'title').should('be', originalTimeStamp);
+
+                    // * Verify the first post timestamp was not modified by the reply
+                    cy.get(`#CENTER_time_${replyId}`).find('time').should('have.attr', 'title').and('not.equal', originalTimeStamp);
+                });
+
+                //# Close RHS
+                cy.closeRHS();
+
+                // * Verify that the reply is in the channel view with matching text
+                cy.get(`#post_${replyId}`).within(() => {
+                    cy.queryByTestId('post-link').should('be.visible').and('have.text', 'Commented on ' + botName + '\'s message: Some Text posted by bot that has no content and no attachment pretext');
+                    cy.get(`#postMessageText_${replyId}`).should('be.visible').and('have.text', 'A reply to an older post bot post');
+                });
             });
         });
 
+        // # Verify RHS is closed
+        cy.get('#rhsContainer').should('not.be.visible');
+    });
+    it('MM-16734 Reply to an older bot post that has some attachment pretext', () => {
```

Comment:
If I test this alone by `it.only`, it failed because it's dependent on some variables (botName, botToken, yesterdaysDate) which were assigned on the first `it` block.  Please check again, thanks